core/domain/platform_parameter_list.py

```diff
@@ -21,25 +21,25 @@
 
 from core.domain import platform_parameter_domain
 from core.domain import platform_parameter_registry as registry
-import utils
+import python_utils
 
 Registry = registry.Registry
 FEATURE_STAGES = platform_parameter_domain.FEATURE_STAGES # pylint: disable=invalid-name
 DATA_TYPES = platform_parameter_domain.DATA_TYPES # pylint: disable=invalid-name
 
-PARAM_NAMES = utils.create_enum( # pylint: disable=invalid-name
+PARAM_NAMES = python_utils.create_enum( # pylint: disable=invalid-name
     'dummy_feature', 'dummy_parameter')
 
 # Platform parameters should all be defined below.
 
 Registry.create_feature_flag(
-    PARAM_NAMES.dummy_feature,
+    PARAM_NAMES.dummy_feature.value,
```

Comment:
Per @vojtechjelinek's comment, ditto here @aasiffaizal. This file has other examples where we should try to preserve the enum type and extract the raw string value later.

Reply:
Will try it out.

Reply:
done